### ISSUE TITLE
Question: Whats the correct way to disable the items dragging on the TreeView control?

### ISSUE THREAD

**Issue by mvegaca** [microsoft/microsoft-ui-xaml#4136]
Hi, I'm Martin from Windows Template Studio Team.

I was working on a treeview page issue on WTS repo.
https://github.com/microsoft/WindowsTemplateStudio/issues/4056

The [TreeView throws an exception when dragging items from different data types](https://github.com/microsoft/microsoft-ui-xaml/issues/4135). I've tried to disable items dragging setting the property **CanDragItems** to false but it didn't work.

As a workaround, we've set **CanReorderItems** property to false.

Is there a known issue here or is this property thought to another purpose?

**Comment by StephenLPeters** [comment]
@chingucoding and @Felix-Dev have either of you ran into this before?

**Comment by chingucoding** [comment]
No, though I haven't used TreeView that much. The fact that CanDragItems gets ignored sounds like a bug to me though.
